README.md

```diff
@@ -1,47 +1,83 @@
+<h1 >Newman</h1>
+<i>The CLI companion for Postman.</i>
+</div>
+<br>
+<p align="center">
+  <a href="https://www.postman.com/"><img src="https://assets.getpostman.com/common-share/postman-logo-horizontal-320x132.png" /></a>
+  <br>
+  <br>
+  <i>Newman is a command-line collection runner for Postman. It allows you to effortlessly run and test a Postman collection directly from the command-line.<br>It is built with extensibility in mind so that you can easily integrate it with your continuous integration servers and build systems.
+  </i>
+  <br>
+  </p>
+<br>
+
+  [![Build Status](https://travis-ci.org/postmanlabs/newman.svg?branch=develop)](https://travis-ci.com/postmanlabs/newman) [![codecov](https://codecov.io/gh/postmanlabs/newman/branch/develop/graph/badge.svg)](https://codecov.io/gh/postmanlabs/newman) 
+![GitHub](https://img.shields.io/github/license/postmanlabs/newman)
+  <hr>
+<br>
+
+  Newman v5 has been released. Check the [migration guide](MIGRATION.md#migrating-from-v4-to-v5) & [changelog](https://github.com/postmanlabs/newman/blob/v5.0.0/CHANGELOG.yaml#L1) for more details.
+
+<hr>
+  <p>
+  <br>
+  <strong>Manage all of your organization's APIs in Postman, with the industry's most complete API development environment.</strong>
+</p>
+
+
+<p align="center">
+  <a href="https://www.postman.com"><strong>www.postman.com</strong></a>
+</p>
+
+<hr>
 
 ## Table of contents
 
-1. [Getting Started](#getting-started)
-2. [Usage](#usage)
-    1. [Using Newman CLI](#using-newman-cli)
-    2. [Using Newman as a Library](#using-newman-as-a-library)
-    3. [Using Reporters with Newman](#using-reporters-with-newman)
-3. [Command Line Options](#command-line-options)
-    1. [newman-options](#newman-options)
-    2. [newman-run](#newman-run-collection-file-source-options)
-    3. [SSL](#ssl)
-    4. [Configuring Proxy](#configuring-proxy)
-4. [API Reference](#api-reference)
-    1. [newman run](#newmanrunoptions-object--callback-function--run-eventemitter)
-    2. [Run summary object](#newmanruncallbackerror-object--summary-object)
-    3. [Events emitted during a collection run](#newmanrunevents)
-5. [Reporters](#reporters)
-    1. [Configuring Reporters](#configuring-reporters)
-    2. [CLI Reporter](#cli-reporter)
-    3. [JSON Reporter](#json-reporter)
-    4. [JUnit Reporter](#junitxml-reporter)
-    5. [HTML Reporter](#html-reporter)
-6. [External Reporters](#external-reporters)
-    1. [Using External Reporters](#using-external-reporters)
-    2. [Creating Your Own Reporter](#creating-your-own-reporter)
-7. [File Uploads](#file-uploads)
-8. [Using Newman with the Postman API](#using-newman-with-the-postman-api)
-9. [Using Newman in Docker](#using-newman-in-docker)
-10. [Using Socks Proxy](#using-socks-proxy)
-11. [Migration Guide](#migration-guide)
-12. [Compatibility](#compatibility)
-13. [Contributing](#contributing)
-14. [Community Support](#community-support)
-15. [License](#license)
+
+  - [Table of contents](#table-of-contents)
```

Comment:
Wondering why it's changed to an unordered list?

Reply:
@codenirvana Does it _need_ to be an ordered list? 

Personally, I feel it looks a lot cleaner without the numbers. 😄